### PR TITLE
some code style tweaks for the FormatNegotiator

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: Negotiation/FormatNegotiator.php [hunk 1]
```diff
@@ -18,26 +18,15 @@
...
 {
-    /**
-     * @var array
-     */
     private $map = [];
-
-    /**
-     * @var RequestStack
-     */
     private $requestStack;
```
Comment (Ener-Getick) [inline Negotiation/FormatNegotiator.php]: Why did you remove this php docs ?

Comment (xabbuh) [inline Negotiation/FormatNegotiator.php]: They did not provide any additional value (the type is already derived by the type hint of the constructor.

Comment (Ener-Getick) [inline Negotiation/FormatNegotiator.php]: I don't know most of the packages always document the properties...
@lsmith77 @stof what do you think ?

Comment (stof) [inline Negotiation/FormatNegotiator.php]: In Symfony (and in my own code), we only document it when the constructor does not allow inferring it. This avoids having to maintain the phpdoc for them (which could become outdated)

Comment (Ener-Getick) [inline Negotiation/FormatNegotiator.php]: Ok, so it's better to remove them when they are useless. Thanks
